unit-tests/sensor/frames/test-filter.cpp

```diff
@@ -0,0 +1,116 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+// We have our own main
```

Comment:
Do we? I don't see it here...

Reply:
Removed

Reply:
Add for CATCH_CONFIG_MAIN: `// Let Catch define its own main() function`
What about CATCH_CONFIG_RUNNER? Why does it need to be defined?